TheMET/TheMET/DataModels/Department.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Department.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 21/03/2023.
+//
+
+import Foundation
+
+class Department: Decodable {
+    
+    let departmentId: Int
+    
+    let displayName: DisplayName
+    
+    let imageURL: URL?
+    
+    let numberOfObjects: Int
+    
+    required init(from decoder: Decoder) throws {
+        <#code#>
+    }
+    
+    enum DisplayName: String {
```

Comment:
This enum doesn't make sense. `Department` has `departmentId` and `displayName` provided by SRV. The amount of departments must not be specified by the App

Reply:
Fixed